lib/views/pages/account_settings/account_settings.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/state/session_provider.dart';
+import 'package:sun_flutter_capstone/views/widgets/template.dart';
+import 'package:sun_flutter_capstone/views/widgets/cards/rectangle.dart';
+
+class AccountSettings extends HookConsumerWidget {
+  const AccountSettings({Key? key}) : super(key: key);
+
+  final String first_name = 'Juan Dela';
+  final String last_name = 'Dela Cruz';
+  final String email = 'juan.delacruz@com.com';
+  final String currency = 'PHP (₱)';
+  final String spending_limit = '18,000';
```

Comment:
According to https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html
```suggestion
  final String firstName = 'Juan Dela';
  final String lastName = 'Dela Cruz';
  final String email = 'juan.delacruz@com.com';
  final String currency = 'PHP (₱)';
  final String spendingLimit = '18,000';
```